lms/static/js/instructor_dashboard/membership.js

```diff
@@ -64,7 +66,7 @@ such that the value can be defined later than this assignment (file load order).
                 item = rowArray[i];
                 $td = $('<td>');
                 if (item instanceof jQuery) {
-                    edx.HtmlUtils.append($td, item);
+                    edx.HtmlUtils.append($td, edx.HtmlUtils.HTML(item));
```

Comment:
Nit: this shouldn't be necessary either as ``HtmlUtils.append`` also safely escapes its input.